### ISSUE TITLE
Symbols imported with wildcard import do not have types enforced

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#5704]
```python
# test1.py
from test2 import *

x: str
reveal_type(x) # int
```

```python
# test2.py
x: int
```

This comes from this mypy issue: https://github.com/python/mypy/issues/10563.

**Comment by erictraut** [comment]
I misdiagnosed the problem. Symbols with a wildcard import can be redefined, but pyright currently doesn't enforce any type inconsistencies for wildcard imports.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.323, which I just published. It will also be included in a future release of pylance.
